Source/Flow/Public/Nodes/Route/FlowNode_Start.h

```diff
@@ -13,6 +13,8 @@ class FLOW_API UFlowNode_Start : public UFlowNode
 {
 	GENERATED_UCLASS_BODY()
 
+	friend class UFlowAsset;
```

Comment:
_Added a friend here so that the UFlowGraph can call "ExecuteInput" rather than "TriggerOutputPin" directly.  It seemed more correct to execute the node rather than trigger its output directly.  The effect is the same for the code as it is now, but if the ExecuteInput has any additional logic added to it, the ExecuteInput would execute it properly, whereas the TriggerOutputPin version would have skipped the additional execute code.

It could be made **public** instead of giving friend access.  Or you we revert the replacements for TriggerOutpinPin -> ExecuteInput and remove the need for the friend access._